### ISSUE TITLE
Migrate Slack channels requiring RSS feed integration

### ISSUE THREAD

**Issue by robrap** [edx/edx-arch-experiments#120]
This task was broken off of https://github.com/edx/edx-arch-experiments/issues/73, which tracked the rest of the Slack migration, which is now complete.

The remaining channels are all blocked for the same reason, and are being tracked in the following spreadsheet: https://docs.google.com/spreadsheets/d/1AmHH1f9L6syrZ190B0GFZ7bUEMQVj6joE2iQ64vjZ8E/edit#gid=0

These channel migrations are blocked on https://jira.2u.com/servicedesk/customer/portal/9/PSO-11955 (Three RSS feed integrations for software releases), created by Tim and shared with Robert and Jeremy (so far). The ticket had a status changed to Prioritized on 11-28-2022, so giving it some time before reaching out again to SaaS Ops.

**Comment by robrap** [comment]
@timmc-edx: Any response?

**Comment by timmc-edx** [comment]
Yes, but there's no good news yet. You should have access to the ticket.

**Comment by robrap** [comment]
@jmbowman: How do you want to proceed?

**Comment by jmbowman** [comment]
Two options come to mind:

* Ask tCRIL if they see value in setting this up for the Open edX Slack workspace, and use it from there.  Most of the packages we want to track are used in Open edX proper.
* It looks like Zapier has some functionality for piping RSS feeds into Slack channels, and we have the Zapier Slack app installed in the 2U workspace.  Is there something self-service we can do to get that working?  https://zapier.com/apps/rss/integrations/slack

**Comment by timmc-edx** [comment]
Is there perhaps a feed aggregator service with 1) an API/OPML upload option and 2) an aggregate feed you can subscribe to?

**Comment by jmbowman** [comment]
Zapier is probably our best option for that if we don't want to mess with self-hosting a service that does it: https://zapier.com/blog/make-your-own-rss-superfeed/ .  Otherwise, there are a handful of paid options like https://www.feedspot.com/rsscombiner/ ($4.79/month if billed annually); it looks like some of the free/inexpensive services for this shut down in the last few years.

**Comment by jristau1984** [comment]
It looks like the spreadsheet is only showing four channels left.  Since it has been over a year, is this still needed?  I see there is a linked ticket in the notes, but I cannot load it to see the status...

>  🛑 Blocked on https://jira.2u.com/servicedesk/customer/portal/9/PSO-11955

I propose closing this ticket as No Longer Needed.

**Comment by timmc-edx** [comment]
I believe Jeremy B was the one primarily interested in this. It also seems like IT is unwilling to help. Given both of those, closing this out.
